packages/panels/src/Pages/Auth/Login.php

```diff
@@ -75,6 +75,14 @@ public function authenticate(): ?LoginResponse
             ]);
         }
 
+        if (! Filament::auth()->user()->canAccessPanel(Filament::getCurrentPanel())) {
+            Filament::auth()->logout();
+
+            throw ValidationException::withMessages([
```

Comment:
Maybe let's extract this throw into its own method `protected function throwFailureValidationException(): void` and then reuse it in both places?